src/ol_infrastructure/applications/mitxonline/Pulumi.applications.mitxonline.CI.yaml

```diff
@@ -6,6 +6,7 @@ config:
 
   mitxonline:db_password:
     secure: v1:vEeGgZVBjWBklqb3:FWrkz0af0N+bAW/Faye1raC5aSL7fJwcJgBk3DtWmQCZ95k1v8pRzMyGmPG6FQ10Q2VWNMS0yBO4PMZVp7bSIRz18OiCPwmfVv5wa347Fms=
+  mitxonline:slack_channel: "kubewatch-applications-ci"
```

Comment:
I routed mitxonline's deployment notifications to this kubewatch test channel for now so we could continue to iterate on the app or webhook if needed.